pkg/kubelet/volume/empty_dir/empty_dir.go

```diff
@@ -19,20 +19,32 @@ package empty_dir
 import (
 	"fmt"
 	"os"
+	"syscall"
 
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubelet/volume"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/types"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util/mount"
 )
 
 // This is the primary entrypoint for volume plugins.
 func ProbeVolumePlugins() []volume.Plugin {
-	return []volume.Plugin{&emptyDirPlugin{nil, false}, &emptyDirPlugin{nil, true}}
+	return ProbeVolumePluginsWithMounter(mount.New())
+}
+
+// ProbePluginsWithMounter is a convenience for testing other plugins which wrap this one.
+//FIXME: alternative: pass mount.Interface to all ProbeVolumePlugins() functions?  Opinions?
```

Comment:
The counter argument is that some volumes (e.g. PD) provide their own Mounter implementation for "real" code, which is counter to real dependency injection.  The reasoning is that GCE provides a "safe format and mount" script, rather than providing a "safe format if needed" script and leaving mount to us.

I decided to stay simple because the alternative is only slightly more consistent and significantly more churn to implement.  I'm happy to be convinced that it's worthwhile, but I'm not there yet